### DISCUSSION TITLE
Custom lsp server config replace whole default configuration.

### DISCUSSION THREAD

**Discussion by JoeMob** [ayamir/nvimdots#1375] [General]
Here's my config of pylsp to enable isort of pylsp
`lua/user/lsp-servers/pylsp.lua`:
`return {
	settings = {
		pylsp = {
			plugins = {
				pyls_isort = { enabled = true },
			},
		},
	},
}`.
Due to the [Steps to modify plugin configs](https://github.com/ayamir/nvimdots/wiki/Usage#steps-to-modify-plugin-configs) part of the wiki, I think the table returned should just update the pyls_isort property but not replace the whole config.

**Comment by ayamir** [comment]
https://github.com/ayamir/nvimdots/pull/1376
